cmd/task/task.go

```diff
@@ -145,6 +144,15 @@ func run() error {
 		return nil
 	}
 
+	if flags.ClearCache {
+		cache, err := taskfile.NewCache(e.TempDir)
```

Comment:
This will conflict with #1661, so we'll need to adapt it here or there, whether PR is merged last.

Reply:
Yes I know, to be honest I would prefer to merge #1661 before then I'll rebase this one.
If it's ok for you

Reply:
Rebase done